tests/performance/decimal_aggregates.xml

```diff
@@ -1,16 +1,13 @@
 <test>
-    <preconditions>
-        <!--FIXME disabled this test until I fix it -->
-        <table_exists>definitely_no_such_table</table_exists>
-    </preconditions>
-
     <settings>
-        <max_memory_usage>20G</max_memory_usage>
+        <max_memory_usage>30G</max_memory_usage>
+        <query_profiler_real_time_period_ns>0</query_profiler_real_time_period_ns>
```

Comment:
But I don't understand why it's needed.

Reply:
I thought that they were unstable because of query profiler:

![scatter-decimal_aggregates-17-client_time](https://user-images.githubusercontent.com/36882414/87659891-e0c7bf00-c766-11ea-86fb-d0e4de675f04.png)
![scatter-decimal_aggregates-17-QueryProfilerSignalOverruns](https://user-images.githubusercontent.com/36882414/87659917-e9b89080-c766-11ea-83ee-bbf7bc37dee7.png)

But it didn't help, they are still unstable.